src/dwarf.cpp

```diff
@@ -187,104 +188,6 @@ std::size_t die_hash(const die& d) {
-    void register_section(const std::string& name, std::size_t offset, std::size_t size);
-
-    void process();
-
-    template <class T>
-    T read();
-
-    std::uint64_t read64();
-    std::uint32_t read32();
-    std::uint32_t read16();
-    std::uint32_t read8();
-    std::uint32_t read_uleb();
-    std::int32_t read_sleb();
-
-    void read_abbreviations();
-    void read_lines();
-    const abbrev& find_abbreviation(std::uint32_t code) const;
-
-    pool_string read_debug_str(std::size_t offset);
-
-    void path_identifier_push();
-    void path_identifier_set(pool_string name);
-    void path_identifier_pop();
-    std::string qualified_symbol_name(const die& d) const;
-
-    attribute process_attribute(const attribute& attr, std::size_t cur_die_offset);
-    attribute_value process_form(const attribute& attr, std::size_t cur_die_offset);
-    attribute_value evaluate_exprloc(std::uint32_t expression_size);
-
-    pool_string die_identifier(const die& a) const;
-
-    die abbreviation_to_die(std::size_t die_address, std::uint32_t abbrev_code);
-
-    pool_string _object_file;
-    freader& _s;
-    const file_details _details;
-    callbacks _callbacks;
-    std::vector<abbrev> _abbreviations;
-    std::vector<pool_string> _path;
-    std::size_t _cu_address{0};
-    std::vector<pool_string> _decl_files;
-    section _debug_abbrev;
-    section _debug_info;
-    section _debug_line;
-    section _debug_str;
-};
-
-/**************************************************************************************************/
-
-template <class T>
-T dwarf::implementation::read() {
-    T result = read_pod<T>(_s);
-    if (_details._needs_byteswap) endian_swap(result);
-    return result;
-}
-
-std::uint64_t dwarf::implementation::read64() { return read<std::uint64_t>(); }
-
-std::uint32_t dwarf::implementation::read32() { return read<std::uint32_t>(); }
-
-std::uint32_t dwarf::implementation::read16() { return read<std::uint16_t>(); }
-
-std::uint32_t dwarf::implementation::read8() { return read<std::uint8_t>(); }
-
-std::uint32_t dwarf::implementation::read_uleb() { return uleb128(_s); }
-
-std::int32_t dwarf::implementation::read_sleb() { return sleb128(_s); }
-
-/**************************************************************************************************/
-
-void dwarf::implementation::register_section(const std::string& name,
-                                             std::size_t offset,
-                                             std::size_t size) {
-    if (name == "__debug_str") {
-        _debug_str = section{ offset, size };
-    } else if (name == "__debug_info") {
-        _debug_info = section{ offset, size };
-    } else if (name == "__debug_abbrev") {
-        _debug_abbrev = section{ offset, size };
-    } else if (name == "__debug_line") {
-        _debug_line = section{ offset, size };
-    }
-}
-
-/**************************************************************************************************/
-
 struct cu_header {
```

Comment:
I moved `cu_header` and `line_header` into the anonymous namespace in this source file, and this is how GH is representing that shift. :eyeroll: